packages/registry/vite-plugin.js

```diff
@@ -14,7 +15,8 @@ export const PloneRegistryVitePlugin = () => {
     { tempInProject: true },
   );
 
-  createAddonsStyleLoader(registry.getAddonStyles());
+  createAddonsStyleLoader(registry);
+  createAddonsLocalesLoader(registry);
```

Comment:
Maybe we should have this also in the init-loaders script, I haven't actually tried yet in Seven so I don't know the answer to this yet.